Testing/Temporary/LastTest.log

```diff
@@ -0,0 +1,3 @@
+Start testing: Nov 01 15:14 PDT
+----------------------------------------------------------
```

Comment:
I assume you did a `git add .`  Please double check that you didn't add any other files by mistake.

Reply:
removed in new commit.